README.md

```diff
@@ -28,6 +28,12 @@ always creates a new error which wraps the original.
  
 You can also add your own additional information.
 
+The stack capturing feature can be turned off for better performance, though it's pretty fast.  Benchmarks
+on an early 2011 MacBook Pro:
+
+    BenchmarkNew_withStackCapture-8     	 1000000	      1428 ns/op
```

Comment:
I assume the 1000000 here is the number of ops/loop size? Should that be clarified?

Also, what does the -8 in the name refer to?